analytics/articles.js

```diff
@@ -72,6 +72,14 @@ if(location.pathname.match('/article/')){
     });
   });
 
+  analyticsHooks.on('impressions:editorial-signup', function(options){
+    analytics.track('Viewed popup for editorial email', {
```

Comment:
The message should mimic these existing article impression analytics: https://github.com/artsy/microgravity/blob/master/analytics/article_impressions.js#L135